### PR TITLE
Add Amazon CodeWhisperer JupyterLab extension

### PR DESCRIPTION
**Pull request by kumarnzt**
*Issue #, if available:*
#116 

*Description of changes:*
Add Amazon CodeWhisperer JupyterLab extension

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kumarnzt** [comment]
Adding new dependency from conda
https://anaconda.org/conda-forge/amazon-codewhisperer-jupyterlab-ext

File: build_artifacts/v1/v1.2/v1.2.0/cpu.env.in [hunk 1]
```diff
@@ -35,3 +35,4 @@ conda-forge::sagemaker-jupyterlab-extension[version='>=0.1.7,<1.0.0']
 conda-forge::sagemaker-jupyterlab-emr-extension[version='>=0.1.7,<1.0.0']
 conda-forge::amazon-sagemaker-jupyter-scheduler[version='>=3.0.2,<4.0.0']
 conda-forge::jupyter-server-proxy[version='>=4.1.0,<5.0.0']
+conda-forge::amazon-codewhisperer-jupyterlab-ext [version='>=2.0.0,<3.0.0']
```
Comment (just4brown) [inline build_artifacts/v1/v1.2/v1.2.0/cpu.env.in:38]: Concerned about the added space character between the name and version bracket. Does this parse successfully for the build?

Comment (kumarnzt) [reply build_artifacts/v1/v1.2/v1.2.0/cpu.env.in:38]: Removing this space.

**Review by claytonparnell — DISMISSED** [review]

**Comment by just4brown** [comment]
Can we include new tests for the package as part of this PR?

**Comment by kumarnzt** [comment]
> Can we include new tests for the package as part of this PR?

Added simple test checking that import works similar to other public extensions.

File: test/test_artifacts/v1/amazon-codewhisperer-jupyterlab-ext.test.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+CMD ["python", "-c", "import amazon_codewhisperer_jupyterlab_ext"]
```
Comment (just4brown) [inline test/test_artifacts/v1/amazon-codewhisperer-jupyterlab-ext.test.Dockerfile:6]: Can you append the new test file to the list of test files in the test harness? https://github.com/aws/sagemaker-distribution/blob/main/test/test_dockerfile_based_harness.py

Ensure both CPU and GPU lists are updated. Thanks

Comment (kumarnzt) [reply test/test_artifacts/v1/amazon-codewhisperer-jupyterlab-ext.test.Dockerfile:6]: > Can you append the new test file to the list of test files in the test harness? https://github.com/aws/sagemaker-distribution/blob/main/test/test_dockerfile_based_harness.py
> 
> Ensure both CPU and GPU lists are updated. Thanks

Added, running the test now. It is taking long.

**Review by claytonparnell — APPROVED** [review]

**Review by just4brown — APPROVED** [review]

**Pull request merged**